spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -759,8 +759,7 @@ def method_missing(sym, *args)
         stub_user(:features => :all)
       end
 
-      ["role_start", "role_suspend", "promote_server", "demote_server",
-       "log_download", "refresh_logs", "log_collect", "log_reload", "logdepot_edit", "processmanager_restart", "refresh_workers"].each do |id|
+      ["role_start", "role_suspend", "promote_server", "demote_server"].each do |id|
```

Comment:
The usage with brackets is seen throughout the whole spec file. It may rather be changed globally using sed. Should it be done as a new branch for standardization?